53 arm subsystem

TheMagneticDude opened the pull request.

TheMagneticDude committed:
Initial Arm subsystem setup

Initial Arm subsystem setup

TheMagneticDude committed:
More setup stuff

More setup stuff

Fyrefyte committed:
revSpark error

TheMagneticDude committed:
Arm levels stuff

Fyrefyte committed:
Consistency again

TheMagneticDude committed:
Merge branch '53-arm-subsystem' of https://github.com/BadRobots1014/BadRobot2023 into 53-arm-subsystem

Fyrefyte committed:
Too many spaces!!!!

Fyrefyte committed:
Merge branch '53-arm-subsystem' of https://github.com/BadRobots1014/BadRobot2023 into 53-arm-subsystem

TheMagneticDude committed:
Arm Preset Positions

JavaBlendPhilippe committed:
Grabber Motor

Added methods for grabber motor

Fyrefyte committed:
Some naming changes

TheMagneticDude committed:
Arm preset positions linked to buttons

linked up buttons

TheMagneticDude committed:
Merge branch '53-arm-subsystem' of https://github.com/BadRobots1014/BadRobot2023 into 53-arm-subsystem

TheMagneticDude committed:
Button arm preset switching no worky

TheMagneticDude committed:
Arm command buttons outline

arm is stuck in stored state

JavaBlendPhilippe committed:
Constants, Constants, and more Constants

Changed all/most of the raw comparators into constants, also added a few spaces here and there. Should be more readable/sound. Its bad practice to compare with raw values across files, constants are safer.

JavaBlendPhilippe committed:
Rudimentary Motor Control

Rudimentary Methods for controlling the motors on the arm. Added the ports.

TheMagneticDude committed:
Arm Preset Positions worky!

JavaBlendPhilippe committed:
Pls use constants nathan i beg of u

TheMagneticDude committed:
Arm high button now moves arm

TheMagneticDude committed:
Button 3 moves arm forward, button 2 moves it backwards and no button kills the motor power

TheMagneticDude committed:
Arm encoder outline

Fyrefyte committed:
Basic extender encoder code

Fyrefyte committed:
More encoder stuff

JavaBlendPhilippe committed:
Grabber Command

Grabber Command

TheMagneticDude committed:
Merge branch '57-arm-encoders' into 53-arm-subsystem

JavaBlendPhilippe committed:
Right Joystick Button

Takes in input to run grabber hopefully

TheMagneticDude committed:
Obliberation

Target Obliberated

TheMagneticDude committed:
Merge branch '53-arm-subsystem' of https://github.com/BadRobots1014/BadRobot2023 into 53-arm-subsystem

JavaBlendPhilippe committed:
Changed Bindings on right controller

TheMagneticDude committed:
Arm encoders

obliberated arm encoders

TheMagneticDude committed:
Merge branch '53-arm-subsystem' of https://github.com/BadRobots1014/BadRobot2023 into 53-arm-subsystem

TheMagneticDude committed:
Obliberated some stuff

Currently the encoders still obloberate themselves when trying to run simulation but if they are fixed run to position should be functional by pressing left trigger.  Currently hard coded to encoder position 5 but I plan on making it changeable using the Z axis roller

TheMagneticDude committed:
Fixed encoder thingys

obloberated encoders

JavaBlendPhilippe committed:
Grabber Overhaul

Overhauled Grabber control commands and implementation. Preset values only, forward and backward. Tested successfully on robot.

TheMagneticDude committed:
obliberated things

TheMagneticDude committed:
Merge branch '53-arm-subsystem' of https://github.com/BadRobots1014/BadRobot2023 into 53-arm-subsystem

TheMagneticDude committed:
Arm Shuffleboard worky now

obliberated shuffleboard

TheMagneticDude committed:
Merge branch 'tank-bot' into 53-arm-subsystem

Kezergan reviewed (commented):
Still some print lines, if that's there accidentally

JavaBlendPhilippe committed:
Grabber Subsystem

JavaBlendPhilippe committed:
Checking current values

JavaBlendPhilippe committed:
Finding Current Output and putting on shuffleboard

Gets the current output, testing is in progress to determine ranges to check amps so it knows when the grabber is filled

Fyrefyte committed:
I fixed the encoders

TheMagneticDude committed:
ARM RUNTOPOSITION WORKYYYYYYY

OBLIBERATED RUNTOPOSITION

Fyrefyte committed:
Fixed resetting encoders

JavaBlendPhilippe committed:
Shuffleboard grabber state

Gets grabber state and returns it. Need to change the logic to make it so it wont reset to empty upon stopping, but will reset if it has ran normally last.

TheMagneticDude committed:
ARM reset encoders worky

obliberated encoders

TheMagneticDude committed:
Obloberated Dunking

DUNK

TheMagneticDude committed:
obliberated preset ticks

obliberated preset ticks

TheMagneticDude committed:
Fixed up down buttons

JavaBlendPhilippe committed:
Grabber States

Grabber States with shufflboard output, needs testing

TheMagneticDude committed:
Added z axis code but does not work in simulation for some reason

Fyrefyte committed:
Cleaned up a lot

Fyrefyte committed:
A little more cleanup

Fyrefyte committed:
These should not be in here

Fyrefyte committed:
Greatly simplified running to positions with arm

Fyrefyte committed:
First test right after this

Fyrefyte committed:
One more change

Fyrefyte committed:
Removed some useless stuff

Fyrefyte committed:
Fixed a BIG issue

Please don't make more Robot Containers it breaks things

Fyrefyte committed:
It dunks now

JavaBlendPhilippe committed:
VEYR IMPORTANT CHANGE

Fyrefyte committed:
Merge pull request #76 from BadRobots1014/centralize-arm-movements

Centralize arm movements

JavaBlendPhilippe committed:
Removed a Redundancy

JavaBlendPhilippe committed:
Merge Fixes

brought in updated arm subsystem, implement grabber code and state getting.

JavaBlendPhilippe committed:
Merge pull request #77 from BadRobots1014/69-grabber-filled-detection

69 grabber filled detection

Fyrefyte committed:
Dee Leat

Fyrefyte committed:
Es ist nicht mehr kaputt

Fyrefyte committed:
Merge branch 'tank-bot' into 53-arm-subsystem

Fyrefyte reviewed (approved):
Itsa okie dokie

JavaBlendPhilippe reviewed (approved):
Lookin Swell *Slightly Crooked Thumbs Up*
